requirements.txt

```diff
@@ -2,3 +2,5 @@ mkdocs==1.2.3
 mkdocs-material==8.1.11
 pymdown-extensions==9.2
 mkdocs-meta-descriptions-plugin==1.0.2
+# Jinja >= 3.1.0 triggers a bug: https://github.com/mkdocs/mkdocs/issues/2799
```

Comment:
Anyways, since it's not the first time that the MkDocs build breaks because of this dependency, I think it's actually better to pin it "permanently". I created a separate pull request for this (https://github.com/codacy/pulse-user-docs/pull/103), so this one doesn't include this mess. :nerd_face: 

```suggestion
```